src/services/types.ts

```diff
@@ -1,14 +1,11 @@
-import { OverrideProperties } from "type-fest";
 import {
   ASTProfile,
-  BridgeFees,
+  ApplicationStatus,
   EndowmentProfile,
   EndowmentProfileUpdate,
   WalletProfile,
 } from "types/aws";
-import { EndowmentDetails } from "types/contracts";
-import { AccountType, EndowmentType } from "types/lists";
-import { Endowment } from "types/subgraph";
+import { EndowmentType } from "types/lists";
```

Comment:
still used in marketplace query params, might be removed on later endpoint versions

Reply:
There's still a need to distinguish types I think, like DAFs, once we're ready for them.